pom.xml

```diff
@@ -28,6 +29,7 @@
         <groupId>org.eclipse.ee4j</groupId>
         <artifactId>project</artifactId>
         <version>1.0.6</version>
+        <relativePath/>
```

Comment:
Why there is empty `<relativePath/>`?

Reply:
make it clear that no check to the default (`../pom.xml`) should be done by maven